meinberlin/apps/budgeting/assets/FilterBar.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
```

Comment:
this file was just copied as base for the Filters, is going to be changed entirely.

Reply:
@khamui i did a reuseable switch component to replace the switch in react but not sure if it'll work with how we want to do it, but we should talk about it before we go further to not double up work https://github.com/liqd/a4-meinberlin/pull/3981/files

Reply:
@phillimorland true! lets use the switch component if possible :) you want to take a look if its possible?
I will focus on the filters then and leave the button untouched. does that make sense?